### ISSUE TITLE
Separate 404 pages from other errors

### ISSUE THREAD

**Issue by nickkolok** [IPTnet/ipt_connect#147]
We should separate 404 pages (removed or non-existing round, etc.) from real errors. Now them both give error 500, which looks not very good for the user and, in addition, creates e-mails to administrators... which are confused with real bugs (so the task is time-saving).

To have an additional portion of fun, I suggest to advertise a competition for the best IPT-related photo for the 404 page. That could be a photo of a moment of a PF (for example, a reporter and an opponent discussing a mission point on reporter's chart) or a photo of preparational process (a team in their lab solving problems). Of course, we need to have the permissions to post the photo from all the people on the photo and from the report's author.

We surely can admit several photos and show them in random order.

Note that I will probably have no time to implement the 404 pages during this tournament, so no hurry with this.  However, advertising the "404 competition" earlier will encourage photographers and players to take the appropriate photos during the fights.

**Comment by nickkolok** [comment]
As an Easter egg: we can use the phrase 'Tactical rejection!' at the 404 page. As an option: use this phrase only if a non-existing problem is requested. 404 errors of this type seem to be extremely rare, so it's a good place to hide the Easter egg :)

**Comment by nickkolok** [comment]
Another idea: we can have zero in "404" number decorated thematically. For example, "0" can be a panel of a tool like voltmeter, but with grades "Found", "Searching..." and "Not Found" and an arrow pointing to "Not Found". Moreover, this "foundmeter" can be an SVG/GIF animation with the arrow first pointing to "Searching..." and then going to "Not Found"

**Comment by david2457** [comment]
I love these ideas! Especially the easter egg hahaha
Also one which Lysenko may be able to help in regards to the design.

**Comment by lysenko-nikolay** [comment]
Yes, I also would like to put .GIF image with T. or P. rejection on this "404" pages.

Questions:
1. could you give me an example of those pages? i have tried smth like [https://connect.iptnet.info/IPT2021](https://connect.iptnet.info/IPT2021) , but it doesn't show "404"
2. What i can think of is:
GIF is made from short video (3-4 sec). Video starts with an empty white wall (1 sec). Paper sheet with some handwriting begins to lift from the bottom (2 sec). When it stops in the middle of the shot we can see the message "TACTICAL REJECTION" in two lines (3 sec). And one second it just stands. 4 sec.
